nixos/modules/services/x11/picom.nix

```diff
@@ -20,39 +20,40 @@ let
 
   mkDefaultAttrs = mapAttrs (n: v: mkDefault v);
 
-  # Basically a tinkered lib.generators.mkKeyValueDefault
-  # It either serializes a top-level definition "key: { values };"
-  # or an expression "key = { values };"
-  mkAttrsString =
-    top:
-    mapAttrsToList (
-      k: v:
-      let
-        sep = if (top && isAttrs v) then ":" else "=";
```

Comment:
Formatting seems to be messed up here. Please remember to format your files with `nixfmt-rfc-style.

`nix-shell -p nixfmt-rfc-style --run "nixfmt nixos/modules/services/x11/picom.nix"`